sap_document_classification_client/http_client_base.py

```diff
@@ -15,6 +16,38 @@
 STATUS_FAILED = 'FAILED'
 
 
+# See: https://requests.readthedocs.io/en/master/user/advanced/#custom-authentication
+class CommonAuth:
+    def __init__(self, url, tenant, secret):
+        self.url = url
+        self.tenant = tenant
+        self.secret = secret
+        self.expires_in = datetime.datetime.now()
+        self.get_access_token()
+
+    def get_access_token(self):
+        if datetime.datetime.now() + datetime.timedelta(seconds=600) > self.expires_in:
```

Comment:
Let's make it 5 seconds, token TTL can also be short in some cases

Reply:
Adopted to 5 seconds & will merge now.
Thanks for review!